### ISSUE TITLE
CLI complains about (0 , _util.promisify) is not a function

### ISSUE THREAD

**Issue by TobbenTM** [olistic/warriorjs#78]
# Environment

## System:
 - OS: Windows 10
 - CPU: x64 Intel(R) Core(TM) i7-3930K CPU @ 3.20GHz
 - Memory: 4.99 GB / 11.94 GB
## Binaries:
 - npm: 4.2.0 - C:\Program Files\nodejs\npm.CMD

# Steps to reproduce

1. Install warriorjs `npm install --global @warriorjs/cli`
2. Run warriorjs `warriorjs`

# Expected Behavior

The CLI works

# Actual Behavior

```
C:\Users\user\AppData\Roaming\npm\node_modules\@warriorjs\cli\lib\PlayerGenerator.js:35
const readFileAsync = (0, _util.promisify)(_fs2.default.readFile);
                                          ^

TypeError: (0 , _util.promisify) is not a function
    at Object.<anonymous> (C:\Users\user\AppData\Roaming\npm\node_modules\@warriorjs\cli\lib\PlayerGenerator.js:35:43)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\user\AppData\Roaming\npm\node_modules\@warriorjs\cli\lib\Game.js:29:24)
    at Module._compile (module.js:571:32)
```

**Comment by TobbenTM** [comment]
Solved by upgrading from node v7.9.0 to v8.11.1, maybe include in requirements?

**Comment by olistic** [comment]
Hello @TobbenTM, thanks for reporting the issue and how you solved it. I'm working on improving node support and I'm going to update the docs as well. Closing this now as there's no more action to be done here.
